app/templates/sell-quick-start.pug

```diff
@@ -4,10 +4,10 @@
       .flex-row.flex-between
         .flex-center
           label.em-500(translate="ENTER_AMOUNT:")
-        .flex-center(ng-show="exchangeRate.fiat" ng-hide="status.fetching")
+        .flex-center(ng-hide="$ctrl.status.fetching")
           span.type-sm.em-500 1 BTC = {{ exchangeRate.fiat }} {{transaction.currency.code}}
           helper-button(content="EXCHANGE_RATE_HELPER")
-        .flex-row.flex-center(ng-show="status.fetching")
+        .flex-row.flex-center(ng-show="$ctrl.status.fetching")
```

Comment:
Yeah, `status.fetching` is still set in `scope.getQuote` and then cleared in the `success` and `error` functions.